packages/manager/modules/models/user/User.class.js

```diff
@@ -31,4 +31,8 @@ export default class User {
   canHaveInvoicesByPostalMail() {
     return this.billingCountry === 'FR' && this.legalform === 'individual';
   }
+
+  isRetractationAllowed() {
```

Comment:
This looks really generic, are we sure it can be applied to something else than telephony product ? 
Probably should use a `isCorporation()` method

Reply:
Good point, name should just reflect what the function does